.gitignore

```diff
@@ -16,3 +16,9 @@
 # Symfony CLI https://symfony.com/doc/current/setup/symfony_server.html#different-php-settings-per-project
 /.php-version
 /php.ini
+.idea/
```

Comment:
it should be in global git ignore on your computer. I think it was somewhere in bitbag bible.